jobs/fashion_mnist_train/job.py

```diff
@@ -66,9 +74,7 @@ def train(project: Optional[str], entity: Optional[str], **kwargs: Any):
         epochs=epochs,
         steps_per_epoch=steps_per_epoch,
         validation_split=0.33,
-        callbacks=[
-            WandbMetricsLogger(),
-        ],
+        callbacks=[WandbMetricsLogger(), WandbModelCheckpoint(filepath="model.h5")],
```

Comment:
Don't we need to save some Adam optimized state as well if we're being truly correct? Is this correct enough?

Reply:
This should happen automatically when saving the model in h5 format https://keras.io/api/saving/model_saving_and_loading/
```
SavedModel format arguments: include_optimizer: Only applied to SavedModel and legacy HDF5 formats. If False, do not save the optimizer state. Defaults to True.
```